programming basics/Python Built-ins.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>Maybe add something like the following to the code...also give the print statements some context:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Lets practice our addition and multiplication
example1 = 5 + 2
print('example1 ', type(example1), example1)

example2 = 6 * 7
print('example2 ', type(example2), example2)

#to force the resultant of 6*7 to be a float, we could take two approaches:
example2b = 6 * float(7)
print('example2b ', type(example2b), example2b)

example2c = 6 *7.0
print('example2c ', type(example2c), example2c)

example3 = 6 / 6.0
print('example3 ', type(example3), example3)


# two asterisks mean raising a number to a power
# in this case, we are squaring the number 7
example4 = 7**2

print('example4 ', type(example4), example4)







</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>